src/nnet3/nnet-am-decodable-simple.cc

```diff
@@ -48,6 +48,15 @@ NnetDecodableBase::NnetDecodableBase(
                  "You need to set the --online-ivector-period option!"));
   log_priors_.ApplyLog();
   PossiblyWarnForFramesPerChunk();
+
+   // extract recurrent output names and their offsets from nnet 
+  std::vector<std::string> recurrent_node_names; 
+  GetRecurrentOutputNodeNames(nnet_, &recurrent_output_names_,
+                              &recurrent_node_names);
+  GetRecurrentNodeOffsets(nnet_, recurrent_node_names, &recurrent_offsets_);
+  for (int32 i = 0; i < recurrent_offsets_.size(); i++)
```

Comment:
just print to the debug log or increase the verbosity for these logging statements.